mautic/mautic
# [UI] Description component

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description
This PR makes descriptions cool.

![image](https://github.com/user-attachments/assets/0f49c1a7-2374-4771-afb2-708b23e8d014)

![image](https://github.com/user-attachments/assets/c66b5efe-7c80-4436-a973-b790fa9e851a)


https://github.com/user-attachments/assets/31e4c23d-e3d0-405f-b3ac-31a26ced206a






| Before                                 | After
| -------------------------------------- | ---
|         ![image](https://github.com/user-attachments/assets/6001ef96-8322-41aa-9b79-967cfaa35ec2)                               | ![image](https://github.com/user-attachments/assets/0f49c1a7-2374-4771-afb2-708b23e8d014)



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Segments 
3. Create new, set its name and a description of less than 70 characters
4. Save & Close
5. See the description is loaded open
6. Create another with more than 71 characters
7. See it's loaded closed by default
